runtime/gc_base/ObjectAccessBarrier.cpp

```diff
@@ -1441,6 +1441,75 @@ MM_ObjectAccessBarrier::cloneIndexableObject(J9VMThread *vmThread, J9IndexableOb
 	return;
 }
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+/**
+ * Copy all of the fields of a value class instance to another value class instance.
+ * The source or destination may be a flattened value within another object, meaning
+ * srcOffset and destOffset need not be equal.
+ * @TODO This does not currently check if the fields that it is reading are volatile.
+ *
+ * @oaram valueClass The value class.
+ * @param srcObject The object containing the value class instance fields being copied.
+ * @param srcOffset The offset of the value class instance fields in srcObject.
+ * @param destValue The object containing the value class instance fields being copied to.
+ * @param destOffset The offset of the value class instance fields in destObject.
+ */
+void
+MM_ObjectAccessBarrier::copyValue(
+							J9VMThread *vmThread, J9Class *valueClass,
```

Comment:
Can you put all of the parameters on one line please?  This will better match the rest of the code in this file.